src/store/site.ts

```diff
@@ -13,3 +14,5 @@ export const siteAtom = atomWithStorage<SiteKey>(
     getOnInit: true,
   },
 )
+
+export const siteIsReloadingAtom = atom(false)
```

Comment:
We need to fully reload the page in some cases - do a full page search on `useReloadOnSiteChange` to see where that applies. This helps us to show the full-page loading indicator while the page is being reloaded by `window.location.reload()`